appium/webdriver/extensions/search_context/android.py

```diff
@@ -24,6 +24,66 @@
 class AndroidSearchContext(BaseSearchContext):
     """Define search context for Android"""
 
+    def find_element_by_android_view_matcher(self, name=None, args=None, className=None):
+        """Finds element by [onView](https://developer.android.com/training/testing/espresso/basics) in Android
+
+        It works with [Espresso Driver](https://github.com/appium/appium-espresso-driver).
+
+        Args:
+            name (:obj:`str`, optional): The name of a method to invoke.
+                The method must return a Hamcrest
+                [Matcher](http://hamcrest.org/JavaHamcrest/javadoc/1.3/org/hamcrest/Matcher.html)
+            args (:obj:`str`, optional): The args provided to the method
+            className (:obj:`str`, optional): The class name that the method is part of (defaults to `org.hamcrest.Matchers`).
+                Can be fully qualified, or simple, and simple defaults to `androidx.test.espresso.matcher` package
```

Comment:
Can be fully qualified by having the `androidx.test.espresso.matcher.` prefix. If the prefix is not provided then it is going to be added implicitly.